Factory/Sources/main.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+#include "computerFactory.h"
+#include "computer.h"
+
+int main()
+{
+	ComputerFactory* computerFactory = new ComputerFactory();
+	IComputer* pc_computer = computerFactory->createComputer("PC");
```

Comment:
Please check the created object for `nullptr` when you have created it via Factory pattern.